servlet/src/main/scala/org/http4s/servlet/DefaultFilter.scala

```diff
@@ -0,0 +1,22 @@
+package org.http4s.servlet
+
+import javax.servlet._
+import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+
+/**
+  * Mainly a convenience for our servlet examples, but, hey, why not.
+  */
+trait DefaultFilter extends Filter {
+  override def init(filterConfig: FilterConfig): Unit = {}
+
+  override def destroy(): Unit = {}
+
+  override def doFilter(request: ServletRequest, response: ServletResponse, chain: FilterChain): Unit =
```

Comment:
Should this method be final? I'm not super familiar with the semantics of using servlet filters.